app/views/users/index.html.erb

```diff
@@ -1,44 +1,18 @@
 <div class="users">
+
     <ul>
+    <% @users.each do |user| %>
+        <a href="<%= user_path(user) %>">
         <li class="user-card">
-        <div class="img-holder">x</div>
-        <div class="card-details">
-            <h4 class="user-name">Name</h4>
-            <p class="num-of-posts">Number of posts</p>
-        </div>
+            <div class="img-holder"><%= image_tag user.photo %></div>
+            <div class="card-details">
+            <h4 class="user-name"><%= user.name %></h4>
+            <p class="num-of-posts">Number of posts: <%= user.post_count %></p>
+            </div>
         </li>
+        </a>
         <br/>
```

Comment:
- [ Required ] I think it would be nice if you center the elements on this page so that the margins on the `left` and `right` sides are equal. Currently, the margins on your design are not equal one is bigger than the other. If you look at the project wireframe for this milestone the margins are equal. Please see the screenshots below and rectify this.

## Your design
![Screenshot from 2023-09-15 02-16-58](https://github.com/danielochuba/duo-blog/assets/16392046/949c2e87-a735-4825-83eb-a2408ab72cfa)

## Project wireframe
![](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=GHSAT0AAAAAABTYBUDZAYGH6RIQCBTWAHBSZIDUK4A)